src/i18n/l10n/en.ftl

```diff
@@ -576,6 +576,18 @@ Uploading-X-Observations = Uploading {$count ->
     *[other] {$count} Observations
 }
 
+# Shows the number of observations a user is currently uploading out of total on my observations page
+Uploading-x-of-y-observations = Uploading {$uploadedCount} of {$total ->
+    [one] 1 observation
+    *[other] {$total} observations
+}
+
+# Shows the progress of uploads for X of Y
+Uploading-x-of-y = Uploading {$uploadedCount} of {$total ->
```

Comment:
You don't need to handle different cases if both counts are displayed as integers.

Reply:
good catch, will fix!